tests/core/test_local_service.py

```diff
@@ -12,38 +12,328 @@
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
 
+import os
+
+import pytest
+from git.repo.base import Repo
+
+from zenml.artifact_stores.base_artifact_store import BaseArtifactStore
 from zenml.artifact_stores.local_artifact_store import LocalArtifactStore
-from zenml.core.local_service import LocalService
+from zenml.core.repo import Repository
+from zenml.metadata.base_metadata_store import BaseMetadataStore
 from zenml.metadata.sqlite_metadata_wrapper import SQLiteMetadataStore
+from zenml.orchestrators.base_orchestrator import BaseOrchestrator
+from zenml.orchestrators.local.local_orchestrator import LocalOrchestrator
 from zenml.stacks.base_stack import BaseStack
 
+LOCAL_STACK_NAME = "local_stack"
+LOCAL_ORCHESTRATOR_NAME = "local_orchestrator"
+LOCAL_METADATA_STORE_NAME = "local_metadata_store"
+LOCAL_ARTIFACT_STORE_NAME = "local_artifact_store"
+
+
+def test_local_service_file_name_exists(tmp_path: str) -> None:
+    """Local Service contains reference to a local service file name."""
+    Repo.init(tmp_path)
+    repo = Repository(str(tmp_path))
+    local_service = repo.get_service()
+    assert local_service.get_serialization_file_name is not None
+
+
+def test_local_service_can_access_metadata_stores(tmp_path: str) -> None:
+    """Local Service can access metadata stores."""
+    Repo.init(tmp_path)
+    repo = Repository(str(tmp_path))
+    local_service = repo.get_service()
+    assert local_service.metadata_stores is not None
+    assert isinstance(local_service.metadata_stores, dict)
+    assert isinstance(
+        local_service.metadata_stores[LOCAL_METADATA_STORE_NAME],
+        BaseMetadataStore,
+    )
+
+
+def test_local_service_can_access_artifact_stores(tmp_path: str) -> None:
+    """Local Service can access artifact stores."""
+    Repo.init(tmp_path)
+    repo = Repository(str(tmp_path))
+    local_service = repo.get_service()
+    assert local_service.artifact_stores is not None
+    assert isinstance(local_service.artifact_stores, dict)
+    assert isinstance(
+        local_service.artifact_stores[LOCAL_ARTIFACT_STORE_NAME],
+        BaseArtifactStore,
+    )
+
+
+def test_local_service_can_access_orchestrators(tmp_path: str) -> None:
+    """Local Service can access orchestrators."""
+    Repo.init(tmp_path)
+    repo = Repository(str(tmp_path))
+    local_service = repo.get_service()
+    assert local_service.orchestrators is not None
+    assert isinstance(local_service.orchestrators, dict)
+    assert isinstance(
+        local_service.orchestrators[LOCAL_ORCHESTRATOR_NAME],
+        BaseOrchestrator,
+    )
+
+
+def test_get_stack_returns_a_stack_when_provided_a_key(tmp_path: str) -> None:
```

Comment:
To me this test seems like it tests the components of the local stack (that it contains the local stores/orchestrators) in addition to checking that a stack is returned, maybe we can incorporate that into the name/docstring somehow